catalog/transformer/catalog-transformer-zip/src/main/java/org/codice/ddf/catalog/transformer/zip/ZipCompression.java

```diff
@@ -63,6 +63,7 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger(ZipCompression.class);
 
+    private static final String METACARD_PATH = "metacards/";
```

Comment:
`METACARD_PATH` is used in two different packages; it seemed wasteful to import the entire package just to use one constant.